packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss

```diff
@@ -0,0 +1,283 @@
+}
+
+&[alignment='bottom-left'] div[role=tooltip]{
+  --cbp-tooltip-placement-top: calc(100% + var(--cbp-tooltip-placement-gap, 5px));
+  --cbp-tooltip-placement-left: calc(0px + var(--cbp-tooltip-placement-offset, 0px));
+  --cbp-tooltip-placement-transform-x: 0;
+  --cbp-tooltip-placement-transform-y: 0;
+  --cbp-tooltip-caret-rotation: 135deg;
+  --cbp-tooltip-caret-left: var(--cbp-tooltip-caret-offset);
+  --cbp-tooltip-caret-top: calc(0% - var(--cbp-tooltip-caret-offset))
+}
+
+&[alignment='bottom-center'] div[role=tooltip]{
+  --cbp-tooltip-placement-top: calc(100% + var(--cbp-tooltip-placement-gap, 5px));
+  --cbp-tooltip-placement-left: 50%;
+  --cbp-tooltip-placement-transform-x: -50%;
+  --cbp-tooltip-placement-transform-y: 0;
+  --cbp-tooltip-caret-rotation: 135deg;
+  --cbp-tooltip-caret-left: calc(50% - var(--cbp-tooltip-caret-offset));
+  --cbp-tooltip-caret-top: calc(0% - var(--cbp-tooltip-caret-offset))
+}
+
+&[alignment='bottom-right'] div[role=tooltip]{
+  --cbp-tooltip-placement-top: calc(100% + var(--cbp-tooltip-placement-gap, 5px));
+  --cbp-tooltip-placement-left: calc(100% + var(--cbp-tooltip-placement-offset, 0px));
+  --cbp-tooltip-placement-transform-x: -100%;
+  --cbp-tooltip-placement-transform-y: 0;
+  --cbp-tooltip-caret-rotation: 135deg;
+  --cbp-tooltip-caret-left: calc(100% - (var(--cbp-tooltip-caret-offset) * 3));
+  --cbp-tooltip-caret-top: calc(0% - var(--cbp-tooltip-caret-offset))
+}
+
+&[alignment='left-top'] div[role=tooltip]{
+  --cbp-tooltip-placement-top: calc(125% + var(--cbp-tooltip-placement-offset, 0px));
+  --cbp-tooltip-placement-left: calc(0% - var(--cbp-tooltip-placement-gap, 5px));
+  --cbp-tooltip-placement-transform-x: -100%;
+  --cbp-tooltip-placement-transform-y: -50%;
+  --cbp-tooltip-caret-rotation: 225deg;
+  --cbp-tooltip-caret-top: calc(25% - var(--cbp-tooltip-caret-offset));
+  --cbp-tooltip-caret-left: calc(100% - var(--cbp-tooltip-caret-offset))
+}
+
+&[alignment='left-center'] div[role=tooltip]{
+  --cbp-tooltip-placement-top: 50%;
+  --cbp-tooltip-placement-left: calc(0% - var(--cbp-tooltip-placement-gap, 5px));
+  --cbp-tooltip-placement-transform-x: -100%;
+  --cbp-tooltip-placement-transform-y: -50%;
+  --cbp-tooltip-caret-rotation: 225deg;
+  --cbp-tooltip-caret-top: calc(50% - var(--cbp-tooltip-caret-offset));
+  --cbp-tooltip-caret-left: calc(100% - var(--cbp-tooltip-caret-offset))
+}
+
+&[alignment='left-bottom'] div[role=tooltip]{
+  --cbp-tooltip-placement-top: calc(-150% + var(--cbp-tooltip-placement-offset, 0px));
+  --cbp-tooltip-placement-left: calc(0% - var(--cbp-tooltip-placement-gap, 5px));
+  --cbp-tooltip-placement-transform-x: -100%;
+  --cbp-tooltip-placement-transform-y: 0%;
+  --cbp-tooltip-caret-rotation: 225deg;
+  --cbp-tooltip-caret-top: calc(75% - var(--cbp-tooltip-caret-offset));
+  --cbp-tooltip-caret-left: calc(100% - var(--cbp-tooltip-caret-offset))
+}
+}
+
+cbp-tooltip div[role=tooltip] {
```

Comment:
This is confusing to me since it's nested inside of the main custom tag, it should be in the same block as the other CSS, not a new block/nesting context. Move it up top before the positioning - it doesn't matter that the placement isn't set until after it. Same for the subsequent blocks starting with `cbp-tooltip`.